src/main/java/sam/Event.java

```diff
@@ -0,0 +1,40 @@
+package sam;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event item that extends the Item class.
+ */
+public class Event extends Item {
+
+    private static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd-MM-yyyy");
```

Comment:
I like how you stored the date format as a constant here! Maybe you can do the same in the "Deadline" class to make it more consistent.